jib-core/src/main/java/com/google/cloud/tools/jib/http/FailoverHttpClient.java

```diff
@@ -263,7 +263,7 @@ public Response call(String httpMethod, URL url, Request request) throws IOExcep
     }
 
     try {
-      return call(httpMethod, url, request, getHttpTransport(true));
+      return call(httpMethod, url, request, getHttpTransport(true), !enableHttpAndInsecureFailover);
```

Comment:
No history for this server, so we enable retrying only when failover is disabled (i.e., if HTTPS fails, then it means failure). OTOH, if failover is enabled, we don't want to retry so that we can quickly failover to test insecure HTTPS.